src/import-export/azext_import_export/vendored_sdks/__init__.py

```diff
@@ -0,0 +1,12 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+
+__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
Is it written by you manually?
Some extensions use the following sentence.
```
__import__('pkg_resources').declare_namespace(__name__)
```

Reply:
should be autogenerated. what's the difference?

Reply:
Reference document is [here](https://packaging.python.org/guides/packaging-namespace-packages/). `__import__('pkg_resources').declare_namespace(__name__)` is no longer recommentded.